package_mrtrix

```diff
@@ -72,7 +64,7 @@ EOF
 
     QTCORE=$(for n in $LIBS; do echo $n; done | grep 'libQt.*Core')
     [[ $QTCORE ]] && ( # sort out Qt:
-      QTPLUGDIR=$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
+      QTPLUGDIR=/usr/lib/qt/plugins/ #$(strings $QTCORE | sed -n 's/qt_plugpath=\(.*\)$/\1/p')
```

Comment:
Sounds good. To be honest, I anticipate it'll only be used by us to prepare binary packages... Eventually I'm hoping neurodebian will pick up the bulk of our Linux packaging load, and this would be used purely to prepare binary releases for those on non-Debian based systems... So not hugely fussed either way.